### ISSUE TITLE
At rules are removed, although they are not detected as errors

### ISSUE THREAD

**Issue by j-a-m-l** [stylelint-scss/stylelint-scss#978]
I've created a repository with a minimal example: https://github.com/j-a-m-l/failed-stylelint-config

So, when I run `npm run check:style`, I don't have any errors related to the usage of any at-rule in:
```sass
@use 'transitions'

@a---asdasf
@lolo

:root
  --header-height: 64px

body
  overflow: hidden
```

but, if I use `npm run fix:style`, all those rules are remove:
```sass
// (empty line)
// (empty line)
:root
  --header-height: 64px

body
  overflow: hidden
```

I've disabled my plugins and presets, but it is still happening.

**Comment by kristerkari** [comment]
@j-a-m-l Sorry for the late answer. 

> I've disabled my plugins and presets, but it is still happening.

This kind of makes it sound like the problem would be related to how stylelint is autofixing the Vue/Sass combo, which is something that I can not really help with since that functionality is part of the code stylelint library offering instead of this collection of rules.

Just to verify, have you check if the stylelint documentation mentions anything about the Sass syntax support? Alternatively you could try using the SCSS syntax with Vue and see if you still run into the same problem.

**Comment by j-a-m-l** [comment]
Thank you. I'm currently focused on other project, so I wouldn't have time to check it soon, but I'll test that when I've finished.
